packages/edit-site/src/components/global-styles/style.scss

```diff
@@ -88,6 +88,12 @@ $block-preview-height: 150px;
 	padding: $grid-unit-20;
 }
 
+.edit-site-screen-background-color__control {
+	.block-editor-color-gradient-control__panel {
+		padding: $grid-unit-20;
+	}
+}
```

Comment:
One small suggestion on the implementation — it would be ideal if we didn't have to rely on an internal CSS classname (`.block-editor-color-gradient-control__panel`) of a `block-editor` component. I think we can determine whether `ColorGradientControl` will be tabbed or not from the logic available in `ScreenBackgroundColor`. Then, you could just apply padding to `.edit-site-screen-background-color__control` itself when `ColorGradientControl` is not tabbed. Does that make sense?

Reply:
> I think we can determine whether ColorGradientControl will be tabbed or not from the logic available in ScreenBackgroundColor. Then, you could just apply padding to .edit-site-screen-background-color__control itself when ColorGradientControl is not tabbed.

Does this mean that when only either the normal or gradient background is active (no tabs), the `ColorGradientControl` is not given a class?

In other words, the code is like this:

```javascript
<ColorGradientControl
	className={
		( ! hasBackgroundColor || ! hasGradientColor ) &&
		'edit-site-screen-background-color__control'
	}
/>
```

Or is it a code like this?

```javascript
<ColorGradientControl
	className={ classNames(
		'.edit-site-screen-background-color__control',
		{
			'has-tab': hasBackgroundColor && hasGradientColor,
		}
	) }
/>
```

Reply:
I have refactored and confirmed that it works as expected as before 👍